desktop/ui/src/main/java/org/datacleaner/cli/CliRunType.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.cli;
+
+public enum CliRunType {
+    DESKTOP, SPARK
```

Comment:
Is this really a "CLI run type" or rather a "execution platform"? I'm thinking that maybe we should replace "DESKTOP" with "LOCAL".

(and maybe in the future there could also be an option like "MONITOR" if one wanted to instrument the DC monitor to run a job there or so.)